english/recite-shakespeare

```diff
@@ -0,0 +1,7 @@
+2nd Monday: This above all: to thine own self be true,
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.